ci/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+use std::ffi::OsString;
+
+use rad_terminal::args::{Args, Help};
+use std::path::PathBuf;
+use url::Host;
+
+pub const HELP: Help = Help {
+    name: "ci",
+    description: env!("CARGO_PKG_DESCRIPTION"),
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+USAGE
+    rad ci [--install/--uninstall] [--seed <host>] [--ssh-key <path>]
+
+OPTIONS
+    --install   Runs docker images for concourse
+    --uninstall Terminates running concourse images
+    --seed      Address of the seed node
+    --ssh-user  SSH user on target machine. Default: root
+    --ssh-index (Optional) Index of SSH key in ssh-agent. Default: 0
+    --config    (Optional) Config file to override defaults
+    --verbose
+"#,
+};
+
+#[derive(Default, Eq, PartialEq)]
+pub struct Options {
+    pub install: bool,
+    pub uninstall: bool,
+    pub seed: Option<Host>,
+    pub ssh_user: String,
+    pub ssh_index: usize,
+    pub config: Option<PathBuf>,
+    pub verbose: bool,
+}
+
+impl Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut install = false;
+        let mut uninstall = false;
+        let mut seed = None;
+        let mut ssh_user = String::from("root");
```

Comment:
It's safer and less surprising to login as the current user and use `sudo` if priviledges are needed.

Reply:
on initial spinner, I now log which user we're ssh'ing using, so if it's the wrong user they can change it https://github.com/radicle-dev/radicle-client-tools/pull/35/commits/21aa4f338dfbf8b7570287a4f7701d4462f48278#diff-2f5300f7dedf306f4b73a94a6c5c3674f3ece69ed4bda39a3b27b8448149d978R122-R126

also added `whoami` https://github.com/radicle-dev/radicle-client-tools/pull/35/commits/21aa4f338dfbf8b7570287a4f7701d4462f48278#diff-2f5300f7dedf306f4b73a94a6c5c3674f3ece69ed4bda39a3b27b8448149d978R58